README.md

```diff
@@ -185,8 +185,8 @@ TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 
-[windows-python]: http://www.python.org/getit/windows
-[windows-python-v2.7.3]: http://www.python.org/download/releases/2.7.3#download
-[msvc2013]: http://www.microsoft.com/en-gb/download/details.aspx?id=44914
-[win7sdk]: http://www.microsoft.com/en-us/download/details.aspx?id=8279
-[compiler update for the Windows SDK 7.1]: http://www.microsoft.com/en-us/download/details.aspx?id=4422
+[windows-python]: https://www.python.org/getit/windows
+[windows-python-v2.7.3]: https://www.python.org/download/releases/2.7.3/#download
```

Comment:
Can you update this to point to 2.7.10?  The URL for that is https://www.python.org/downloads/release/python-2710/ I think.